pkgs/dart_flutter_team_lints/CHANGELOG.md

```diff
@@ -1,6 +1,8 @@
-## 3.1.1-wip
+## 3.2.0
 
-- Contribute a (brief) change policy to the readme.
+- Contributed a (brief) change policy to the readme.
+- Added `invalid_runtime_check_with_js_interop_types`.
```

Comment:
I think it's good to add this lint, but I imagine this being in a minor release might break CI for a few repos, as I found quite a few cases of this lint being hit.

That might be a good thing though, as it's almost always indicating an error!

Reply:
I think it's ok to add new lints in a minor version for this package; we've made that call before. It should only break our own repos and we can update as we see failures. It also lets us dogfood earlier - otherwise we'd have to send PRs to each repo to rev the version constraint before we saw the new version.

> ... as I found quite a few cases of this lint being hit

That's good data - we were wondering how much this would trigger on existing code.